src/Message/CIMResponse.php

```diff
@@ -7,6 +7,11 @@
  */
 class CIMResponse extends CIMAbstractResponse
 {
+    /**
+     * For Error codes: @see https://developer.authorize.net/api/reference/responseCodes.html
+     */
+    const ERROR_RESPONSE_CODE_CANNOT_ISSUE_CREDIT = 54;
```

Comment:
>  other error codes are in string format
Which other error codes?

I think the typecasting its not a big deal. I rather keep error codes as constants and keep this pattern going (its been done in CS-137 too). Cleaner and readable code.